### ISSUE TITLE
Skill Key Word

### ISSUE THREAD

**Issue by zerosistem** [Treeofsavior/ThaiTranslation#20]
เพื่อให้นักแปลทั้งหลายได้เข้าใจตรงกันและแปลไปในทิศทางเดียวกัน
อีกทั้งจะได้สะดวกต่อการค้นหาและแก้ไข
อยากจะให้ช่วยกันกำหนดคำแปลที่ชัดเจนจะได้สะดวกต่อการทำงาน

We should give definition of the important keywords.
For the mutual understanding of our translators.
To work more easily or find and replace translated words.

ใครเจอคำศัพท์หลักๆอะไร หรืออยากให้ปรับปรุงแก้ไขคำไหนก็มาคุยกันในนี้จะได้แปลเหมือนๆกัน
จะได้สะดวกกับคนอื่นๆที่อาจจะมาทำงานต่อไปด้วย

**Comment by zerosistem** [comment]
Key word จาก Skill_ENG
Physical - กายภาพ
Range - ระยะไกล
Magic - เวทมนตร์

Attack - โจมตี
Buff - เพิ่มค่าสถานะ // (เสริมพลัง)(เผื่อไว้แก้ในกรณีที่พบสกิลลดค่าสถานะแต่เพิ่มความสามารถ)
Debuff - ลดค่าสถานะ // (บั่นทอนกำลัง)
Skill - สกิล
Curse - สาป
Block - 
Summon - อัญเชิญ
Monster - สัตว์ประหลาด
XX times - XX ครั้ง
XX hits - XX ที
Damage - ความเสียหาย

Strike - ฟาด
Pierce - แทง
Slash - ฟัน
Blow - กระแทก

Dash - พุ่ง
Knockback - กระเด็น
Smash - ทุบ
Smack - ตบ
Stomp - กระทืบ
Tramp,Trample - เหยียบ

Basic - พื้นฐาน
Body Attack - โจมตีลำตัว
Body Slam -ใช้ตัวกระแทก
Continuous Attack - โจมตีต่อเนื่อง
Fall Attack
Fume Flame - พ่นไฟ
Fume Smoke - พ่นควัน
Head butt - โขก
Normal Attack - โจมตีปกติ
Rotation,Rotating,Rotate Attack - หมุนตัวโจมตี
Spread poison - ปล่อยพิษ
Tail whip - ฟาดหาง
Two-handed Attack - โจมตีด้วยสองมือ
Two-handed clap - ประกบมือ
Two-handed Strike - ฟาดด้วยสองมือ
Item skill

non-Property - ไร้ธาตุ
Fire Property - ธาตุไฟ
Ice Property - ธาตุน้ำแข็ง
Water Property - ธาตุน้ำ
Dark Property - ธาตุมืด
Holy Property - ธาตุศักดิ์สิทธิ์
Poison Property - ธาตุพิษ
Lightning Property - ธาตุสายฟ้า
Earth Property - ธาตุดิน
Non-Attribute

Reflect Shield - เกราะสะท้อน
Claymore Explosion - ระเบิดเคลย์มอร์
Wall of Flame - กำแพงไฟ
Detonate - จุดชนวน
Mine - กับระเบิด
Ice Pike
Fire Pillar
Fire ball
Heal
Heal wind
Heal Floor
Floor Skill
Energy Bolt
Ice Bolt
Refreezer Wave
Round Target
Roar - คำราม
Transforce

**Comment by phone1246** [comment]
ชื่อกิล ถ้าแปลมันจะแปลกๆปะครับบางอัน

**Comment by HalfEmptyChair** [comment]
non-Property เสนอให้เปลี่ยนเป็น 'ไร้ธาตุ' แทนครับ มันดูกระชับกว่า

ส่วนชื่อสกิล ทั้งของผู้เล่นและของศัตรู แนะนำให้ใช้เป็นภาษาอังกฤษไปดีกว่าครับ แปลแค่คำอธิบายสกิลก็พอ

**Comment by Tampzen** [comment]
ชื่อเมืองควรเปลี่ยนไหมครับ ส่วนตัวผมว่าให้ใช้เป็นภาษาอังกฤษน่าจะดี
